src/multiparser.js

```diff
@@ -91,11 +140,40 @@ function fromBabylonFlowOrTypeScript(path) {
         };
       }
 
+      /**
```

Comment:
This will be the first time it is possible to infinitely nest the multiparser...

Reply:
```js
markdown`
     ```js
     markdown`
          ```js
          console.log('hi');
          ```
     `
     ```
`
```

Reply:
Oh, I think we should use `~~~`-style code block for markdown in template string.

Reply:
I didn't even know that existed!